Exec/gravity_tests/code_comp/Castro_ext_src.H

```diff
@@ -0,0 +1,33 @@
+#ifndef CASTRO_EXT_SRC_H
+#define CASTRO_EXT_SRC_H
+
+#include <prob_parameters.H>
+
+using namespace amrex;
+
+AMREX_GPU_DEVICE AMREX_FORCE_INLINE
+void do_ext_src(int i, int j, int k,
+                GeometryData const& geomdata, 
+                Array4<const Real> const& state,
+                Array4<Real> const& src,
+                GpuArray<Real, 3> center,
```

Comment:
should center be a reference? or does this not matter?

Reply:
I had intended this to be a reference, but checking AMReX there are a few examples of GpuArrays not being passed by reference to functions, so maybe it's not necessary? I'll switch it to be passed by reference anyway.

Reply:
I don't know which is preferred here.  I recall once deciding that we should pass these all by reference, but since this will be inlined, maybe it doesn't matter anyway.